include/zephyr/net/prometheus/collector.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024 Mustafa Abdullah Kus, Sparse Technology
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_PROMETHEUS_COLLECTOR_H_
+#define ZEPHYR_INCLUDE_PROMETHEUS_COLLECTOR_H_
+
+/**
+ * @file
+ *
+ * @brief Prometheus collector APIs.
+ *
+ * @defgroup prometheus Prometheus API
+ * @since 3.8.0
```

Comment:
I checked the rendered documentation, and it seems this should be written as `@since 3.8` as now the doc output says `v3.8.0.0` which is not correct.
See https://builds.zephyrproject.io/zephyr/pr/72804/docs/develop/api/overview.html

Reply:
Fixed it. That issue related one file and update it as 3.8.